### DISCUSSION TITLE
프로젝트 세팅 옵션 변경 적용

### DISCUSSION THREAD

**Discussion by xowjd913** [naverz/zepeto-studio-kor#802] [5. Q&A]
프로젝트 세팅의 퀄리티 변경이 QR 코드 빌드 및 퍼블리시 빌드에 적용이 되나요 ?
현재 Realtime Reflection Probes 를 사용하려는데 적용이 안되고있습니다 ㅠㅠ.. (빌드 후 모바일에서 확인 시)

Bake 해서 들어가는건 정상작동 하고있습니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 
기본적으로 월드의 퀄리티는 제페토 앱의 퀄리티 설정을 따르도록 되어 있습니다.
월드 개별적으로 퀄리티를 다르게 하는 추가 기능 개발 이슈이며 다음 마일스톤 목표로 진행 중입니다.
감사합니다.
